app/views/records/edit_fields/_date_created.html.erb

```diff
@@ -1,4 +1,9 @@
 <div class="form-group optional generic_file_identifier">
   <%= f.label :date_created %>
-  <%= f.date_field :date_created, class: 'form-control', name: "generic_file[date_created][]", value: "#{@generic_file.date_created.first if @generic_file}" %>
+  <%= f.date_field :date_created,
+                   class: 'form-control',
```

Comment:
No, that's me.  I lined up all of the params.  Otherwise Hound would start complaining.